app/views/posts/show.html.erb

```diff
@@ -1,9 +1,29 @@
 <%= stylesheet_link_tag 'user', media: 'all', 'data-turbolinks-track': 'reload' %>
 <%= stylesheet_link_tag 'post', media: 'all', 'data-turbolinks-track': 'reload' %>
+
+<body>
+<% flash.each do |key, value| %>
+  <div class="flash <%= key %>">
+    <%= value %>
+  </div>
+<% end %>
+<nav class="nav">
+    <%= link_to "Home", root_path %>
+    <%= link_to "Back", :back %>
+</nav>
+
 <%= render partial: 'post', locals: { post: @post } %> <br>
 <%= "no comments for this post" if @post.comments.empty?%>
 <div class="comments">
 <% @post.comments.each do |comment|%>
         <%=render comment %>
 <% end %>
-</div>
\ No newline at end of file
+</div>
+
+<div class="like-comment"><%= render "comments/create" %> 
+<%if Like.where(author_id: User.first.id, post_id: @post.id).exists?%>
+ <p id= "liked">Liked<p>
+<%else%>
+<%= render "likes/create" %></div>
+<%end%>
```

Comment:
- [ ] An error occurs when attempting to create a new post and it has to do with a missing partial `likes/_create`as shown below 👇. Please try to fix this error 🐛 
![image](https://github.com/Peter-Dumbari/Blog-Application/assets/46839548/64146608-d5e7-4a9c-8dbc-e326eb790bf6)

Reply:
@codepantha, sorry how are you starting the app?